SitecorePackageDeployer/packages.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="4.5.9" targetFramework="net45" />
-  <package id="SitecoreKernel" version="7.0.0.131127" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="net462" />
+  <package id="Sitecore.Kernel.NoReferences" version="8.2.171121" targetFramework="net462" developmentDependency="true" />
```

Comment:
Interesting... so 8.2.171121 depends on 4.6.2? That seems strange. Again, I looked at a vanilla install of 8.2u5 (maybe one behind 171121?) and it targets 4.5.

Reply:
Check out Note 23 here.  Going to drop it down to 4.6.1, though.

Reply:
https://kb.sitecore.net/articles/087164#notes